modules/ROOT/pages/dw-operators.adoc

```diff
@@ -363,3 +363,359 @@ xref:dataweave-variables.adoc#example_local_variable[Examples: Local DataWeave V
 | `if else` a| An `if` operator evaluates a conditional expression and returns the value under the `if` only if the conditional expression is true. Otherwise, it returns the expression under `else`. Every `if` expression must have a matching `else` expression. See xref:dataweave-flow-control.adoc#control_flow_if_else[if else] for an example.
 | `else if` | An `else` operator chains expressions together within an if-else construct by incorporating `else if`. See xref:dataweave-flow-control.adoc#control_flow_else_if[else if] for an example.
 |===
+
+== Update Operator
+
+DataWeave 2.0 supports the update operator that enables you to update a given field on a data structure with a new value. This use case is very common and was not simple to do in DataWeave.
```

Comment:
```suggestion
DataWeave supports the `update` operator, which enables you to update a given field of a data structure with a new value.
```

Reply:
Also, the very first paragraph of the page should probably mention something about this functionality. "DataWeave 2.0 supports several mathematical, equality, relational, logical, prepend, append, **flow control, scope, and update** operators."

Reply:
`a given field` -> should we say `specified fields` to make it clear several can be modified at once? @machaval

Reply:
Updated.